politeiawww/config.go

```diff
@@ -620,21 +620,25 @@ func loadConfig() (*config, []string, error) {
 		log.Warnf("%v", configFileError)
 	}
 
-	newUserTplData := newUserEmailTemplateData{
-		Email: "test@example.com",
-		Link:  "http://www.example.com",
-	}
-	if err := templateNewUserEmail.Execute(os.Stdout, &newUserTplData); err != nil {
+	if err := loadEmailTemplates(); err != nil {
 		return nil, nil, err
 	}
 
-	resetPasswordTplData := resetPasswordEmailTemplateData{
-		Email: "test@example.com",
-		Link:  "http://www.example.com",
+	return &cfg, remainingArgs, nil
+}
+
+func loadEmailTemplates() error {
+	asset, err := Asset(filepath.Join(defaultTemplateDir, "newuser.html"))
```

Comment:
You need to use the same logic as, for example, defaultLogDir.  Find all instances of the log stuff and copy the logic for templates.  This needs to become a setting as well in the .conf + flags.